WORKSPACE

```diff
@@ -1,4 +1,9 @@
 workspace(name = "io_bazel_rules_appengine")
 
-load("//appengine:appengine.bzl", "appengine_repositories")
-appengine_repositories()
+load("//appengine:java_appengine.bzl", "java_appengine_repositories")
```

Comment:
prefer to group loads first

Reply:
Done. It would be nice if buildifier did that for me.